apps/meteor/app/models/server/models/LivechatVisitors.js

```diff
@@ -13,6 +13,7 @@ export class LivechatVisitors extends Base {
 		this.tryEnsureIndex({ 'visitorEmails.address': 1 }, { sparse: true });
 		this.tryEnsureIndex({ name: 1 }, { sparse: true });
 		this.tryEnsureIndex({ username: 1 });
+		this.tryEnsureIndex({ contactManager: 1 }, { sparse: true });
```

Comment:
Instead of an index on full contactManager obj, try with an index on "contactManager.username" as it's the used prop 👀

Reply:
makes sense, doing it.